### DISCUSSION TITLE
Expand ImageSharp.Web Support for Background Color?

### DISCUSSION THREAD

**Discussion by jeffolmstead** [OrchardCMS/OrchardCore#7584] [Ideas]
Right now, when I use a media resize_url to generate an image using pad as a jpg, then I am left with a black background where the padding is done. It looks like [ImageSharp.Web supports bgcolor](https://docs.sixlabors.com/articles/imagesharp.web/processingcommands.html#background-color) but I am not super familiar with the library. Would it be possible to [expand the Orchard Core code](https://github.com/OrchardCMS/OrchardCore/blob/c46579c150f7bec0e836161bcec6abe40406791d/src/OrchardCore.Modules/OrchardCore.Media/Processing/ImageSharpUrlFormatter.cs#L31) and add in the option for passing in bgcolor? Then this could be exposed in Liquid as well to utilize. I feel that would allow me to then use jpg more often (rather than the png I am using right now - they are large files).

**Comment by deanmarcussen** [comment]
Can you open an issue @jeffolmstead ?

ImageSharp.Web does support bgcolor, we have just always removed it (probably due to the potential for abuse)

With token support I'm happy to see that support added as well, so a pr is welcome.

**Comment by netwavebe** [comment]
https://github.com/OrchardCMS/OrchardCore/pull/7647
